### PR TITLE
Fix typos in Section 5.4

### PR DESCRIPTION
**Pull request by tobolar**

### CODE REVIEW

**Review by HansOlsson — APPROVED** [review]
Looks good.

File: chapters/scoping.tex [hunk 1]
```diff
@@ -317,7 +317,7 @@ \section{Simultaneous Inner/Outer Declarations}\label{simultaneous-inner-outer-d
   der(x) = if isEnabled then -x else 0;
 end ConditionalIntegrator;
 
-model SubSystem "subsystem that 'enable' its conditional integrators"
```
Comment (henrikt-ma) [inline chapters/scoping.tex:320]: This has the same grammatical error that was corrected above: 'enable' -> 'enables'

Comment (HansOlsson) [reply chapters/scoping.tex:320]: I believe the apostrophes should be removed so 'enable' -> enables.
It seems they are used to indicate Modelica code in comments below, but it doesn't make sense in this place as there is no variable named enable.

Comment (tobolar) [reply chapters/scoping.tex:320]: @HansOlsson I have additionally pushed to this PR but is was merged before. How can I restart this PR? Or shall I create a new PR?

Comment (HansOlsson) [reply chapters/scoping.tex:320]: A new PR is likely simplest.

**Pull request merged**